### DISCUSSION TITLE
Manual/webgl-building

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#513] [Notes]
# Manual/webgl-building

https://docs.unity3d.com/Manual/webgl-building.html

**Comment by radiatoryang** [comment]
Note that the dual build texture compression code example is wrong, you have to set the WebGLTextureSubtarget a different (undocumented) way:

```
BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
buildPlayerOptions.subtarget = (int)WebGLTextureSubtarget.ASTC;
```
